frontend/public/style/_common.scss

```diff
@@ -170,6 +170,11 @@ dl.co-inline {
   margin: 15px 0;
 }
 
+.co-switch-label .pf-c-switch__label {
+  font-size: $font-size-base;
+  font-weight: var(--pf-global--FontWeight--normal); // override `label` style from _bootstrap-residual.scss
+}
```

Comment:
This is not good. We should not look to override anything in PF... if this is because the designs request it, we need to push back on UX and inform them that PF default styles is the styles we use. We shouldn't override components to match a use-case.

Reply:
This is the result of existing console styles that are out of sync with PF4.  Until we can get the entire console updated to vanilla PF4, they're a necessary evil.

Reply:
Alright -- if that's the case, can we put a comment to that extent with like a `TODO: remove when x is completed`.

The knowledge of why we are doing this is lost on me, so I'm sure the next person to see this might have a similar confusion point.